### ISSUE TITLE
Elastic Search not working and is behaving as CRUD module search too

### ISSUE THREAD

**Issue by mardo** [feliperazeek/playframework-elasticsearch#12]
Steps to reproduce the issue:

1)Please find the ZIP file in the next link and uncompress it:
http://sivesagt.com/stuff/elasticSearchProblem.zip

2)Navigate to the uncompressed folder and from a terminal hit: 
play run

3)Open localhost:9000 on your browser

4)At the header menu we now can find 3 links:
"CRUD search(should NOT return results)",
"Elastic search(should NOT return results)" and
"Elastic search(should only return ONE result)"

5)Follow both Elastic Search links and find that the searching is not behaving as it should. (the other link, CRUD module search, is not an issue for this module, is just there to show that the searching is behaving the same way as this module) 

Additional information:
Basically each link all it does is put strings on the search input boxes

**Comment by mardo** [comment]
Today i did another test:
- Created a Model with @ElasticSearchable
- Inserted a record to the database directly using MySQL Query Browser (so it wont create an index)
- Verified that there are no indexes created by elasticsearch
- Created a ElasticSearchController for the Model
- Searched for a text AND I COULD FOUND MY RECORD without any indexes in ElasticSearch

I think there is no difference by searching via ElasticSearchController or CRUD module. Also i think searching this way is not using Lucene for this.

Can you check this please?
Thank you!

P.D. I also found a very very small issue in elasticsearch 0.0.8, in the tag pagination.html you are using:
controllers.ElasticSearchController
The application crashes because is not the correct package.
i changed to controllers.elasticsearch.ElasticSearchController and its working

**Comment by bgooren** [comment]
Currently the ElasticSearchController performs a (JPA) database search, not an elastic search. I'm currently trying to fix this, but it'll take some effort.

**Comment by bgooren** [comment]
I'm about to issue a pull request which fixes this

**Comment by mardo** [comment]
Thank you very much bgooren!

**Comment by bgooren** [comment]
Done
